app/js/populateTestDropdowns.ts

```diff
@@ -15,6 +15,9 @@ function populateTestDropdowns () {
             let template: Function = Handlebars.compile(HBTemplate);
             document.querySelector<HTMLElement>('#test_id').innerHTML = template(testsObject);
 
+            document.querySelector<HTMLElement>('#linkToEditTestsBtn').setAttribute('href', `editTests.html#${testsObject.data[0].id}`);
```

Comment:
Where did this line come from? Seems to be unused and is just triggering an error.